### ISSUE TITLE
Split tokens by chain ID

### ISSUE THREAD

**Issue by moodysalem** [Uniswap/tokenlists-org#1081]
Token lists will now often have tokens on different chains (rather than just mainnet)

We should split tokens by chain ID in the display

**Comment by masonicGIT** [comment]
Yes. Would be good to see if there are any thoughts around making this work for non-EVM solutions as well in the long term.

One potential solution might be to set a chain ID for non-EVM based solutions, so there is some sort of known and agreed upon mapping.

**Comment by wbnns** [comment]
> Token lists will now often have tokens on different chains (rather than just mainnet)
>
>We should split tokens by chain ID in the display, maybe showing names for known network chain IDs

Agreed, this would be a nice enhancement and make it easier to organize assets across networks.

**Comment by ef3n9r98** [comment]
Hey @willhennessy,

Have you found a developer for this yet? 

If not, I can add it to [our site](https://www.openq.dev) and then find a dev for you.

You wouldn’t have to fund a bounty until someone wants to fix it.

Your bounty is locked in a smart contract until you merge the dev’s PR. That releases it automatically.

Let me know if you have a budget and I’ll add it to our site.
